src/main/java/net/mcreator/element/types/VillagerTrade.java

```diff
@@ -63,4 +63,18 @@ public static class Entry {
 			public double priceMultiplier;
 		}
 	}
+
+	public boolean hasWanderingTrade() {
+		for (CustomTradeEntry tradeEntry : tradeEntries) {
+			if ("WanderingTrader".equals(tradeEntry.villagerProfession.getUnmappedValue())) return true;
```

Comment:
`"WanderingTrader"` is mapped value, not unmapped, does this still work?
Also, the return statement here should be located on a separate line to follow the code style.

Same for the other method.